api/admin.php

```diff
@@ -36,11 +36,16 @@
 }
 
 if ($data['type'] == 'config') {
-    $file = __DIR__ . '/../config/my.config.inc.php';
     $newConfig = [];
 
     foreach ($config as $k=>$conf) {
         if (is_array($conf)) {
+
+            if (isset($data[$k]) && !empty($data[$k]) && is_array($data[$k])) {

```

Comment:
If we use `empty`, we don't have to use `isset`.

Reply:
Fixed.